backend/src/packages/surveys/libs/types/types.ts

```diff
@@ -1,4 +1,20 @@
+import { type SurveyEntity } from '#packages/surveys/survey.entity.js';
+
+type SurveyCreateResponseDto = {
```

Comment:
should be in separate file, not in file for re-export

Reply:
done!